### ISSUE TITLE
unable to pull latest 4.7.11 docker image

### ISSUE THREAD

**Issue by colinerd** [firefly-iii/firefly-iii#2073]
I am running Firefly III version 4.7.10

**Description**
trying to update to 4.7.11 but when pulling the latest tag, I keep getting the 4.7.10 image from a few days ago.

**Extra info**
docker installation on unraid, can be managed by unraid and portainer. trying to pull jc5x/firefly-iii:latest but keep getting the 4.7.10 image from february 3rd.

on dockerhub, it is clearly visible the latest tag was updated 4 hours ago.

for 4.7.10 there was a regular tag ‘release-4.7.10’ available but I am not seeing that for 4.7.11, only the arm, amd and amd64 tags for 4.7.11 are visible.

**Comment by JC5** [comment]
For some reason the master build failed yesterday. I’ll have to do some debugging and releas it manually later.

**Comment by JC5** [comment]
I've pushed the "latest" tag again, could you try it?

**Comment by JC5** [comment]
I've also pushed a new manifest for tag release-4.7.11

**Comment by colinerd** [comment]
Hi,
Tried it on 2 seperate docker environments (even new clean install) but the latest image always results in some access rights issue.

Unable to create the cache directory (/var/www/firefly-iii/storage/framework/views/twig/e3).

It is writing other files and folders in the volume so not sure why it is not able to create the cache directory.

Another interesting note is that the image being pulled (both latest & release-4.7.11) are not from 11/02/2019 but from 10/02/2019.

![afbeelding](https://user-images.githubusercontent.com/15741333/52667914-71ff8a00-2f12-11e9-88a7-458364fd25c9.png)

![afbeelding](https://user-images.githubusercontent.com/15741333/52667975-965b6680-2f12-11e9-9ea0-7bcf5e6782b6.png)

Just to try; I did pull the develop image that I can see for today and that one works fine and the date is matching as well.

![afbeelding](https://user-images.githubusercontent.com/15741333/52668134-fa7e2a80-2f12-11e9-931e-b13e1e0e6f4a.png)
